version.sh

```diff
@@ -0,0 +1 @@
+MKLIVE_VERSION="0.23 $(git rev-parse --short HEAD)"
```

Comment:
i think having the rev-parse happen at runtime will not work in installer.sh when it's included in a live image

Reply:
maybe there could still be a sed to replace the version in the installer done by make-x86-images

Reply:
good catch, fixed, this now depends on #300